scripts/packages/VSCodeServer/src/trees.jl

```diff
@@ -198,8 +198,8 @@ function getvariables()
             tree.canshow = can_display(x)
             push!(variables, tree)
         catch err
-            printstyled("Internal Error: ", bold = true, color = Base.error_color())
-            Base.display_error(err, catch_backtrace())
+            # printstyled("Internal Error: ", bold = true, color = Base.error_color())
+            # Base.display_error(err, catch_backtrace())
```

Comment:
Maybe attach the link to [the issue](https://github.com/julia-vscode/julia-vscode/issues/1491) here for the future reference ?